### PR TITLE
Add Example.cs

### PR DESCRIPTION
**Pull request by naveen965**

### CODE REVIEW

File: Assets/Scripts/Example.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+public class Example : MonoBehaviour
```
Comment (naveen965) [inline Assets/Scripts/Example.cs:3]:
```suggestion
public class Path : MonoBehaviour
```

**Review by naveen965 — COMMENTED** [review]
I will changed mentioned line

File: Assets/Scripts/Example.cs (hunk 1 above)
Comment (senaveen5) [reply Assets/Scripts/Example.cs:3]: Yes. this is good

**Review by senaveen5 — APPROVED** [review]
Approved

**Pull request merged**